features/telemetry/protocols/netflow/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/common/NetflowMessage.java

```diff
@@ -110,7 +110,7 @@ public Integer getEngineType() {
 
     @Override
     public Long getDeltaSwitched() {
-        return flowMessageProto.hasDeltaSwitched() ? flowMessageProto.getDeltaSwitched().getValue() : null;
+        return flowMessageProto.hasDeltaSwitched() ? new Long(flowMessageProto.getDeltaSwitched().getValue()) : getFirstSwitched();
```

Comment:
I don't think the extra boxing with `new Long` is necessary here.

Reply:
`ternary` operator tries to auto unbox null and encounters NPE.  WIth boxing the first expression it doesn't unbox the second expression.